docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -0,0 +1,314 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/getting-started/migrating-from-static/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/getting-started/migrating-from-static/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/getting-started/migrating-from-static/
+canonical: https://grafana.com/docs/agent/latest/flow/getting-started/migrating-from-static/
+description: Learn how to migrate your configuration from Grafana Agent Static Mode to Grafana Agent Flow Mode
+  Flow Mode
+menuTitle: Migrate from Grafana Agent Static Mode
+title: Migrate from Grafana Agent Static to Grafana Agent Flow Mode
+weight: 340
+---
+
+# Migrate from Grafana Agent Static to Grafana Agent Flow Mode
+
+The built-in Grafana Agent convert command can migrate your [Static][]
```

Comment:
There a lot of places I say [Static] or Grafana Agent Static Mode or variances. Please feel free to correct these to what they should be in each case.